### PR TITLE
add keras2 Core layers(Activation,Dropout,Flatten)

### PR DESCRIPTION
**Pull request by yjjiang11**

### CODE REVIEW

**Comment by yjjiang11** [comment]
Jenkins passed:http://172.168.2.101:8080/job/ZOO-PR-Scala-Spark-2.1/776/

**Comment by yjjiang11** [comment]
Jenkins passed: http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/775/

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras2/layers/Dropout.scala [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import scala.reflect.ClassTag
+
+/**
+ * Applies Dropout to the input by randomly setting a fraction 'rate' of input units to 0 at each
+ * update during training time in order to prevent overfitting.
+ *
+ * When you use this layer as the first layer of a model, you need to provide the argument
+ * inputShape (a Single Shape, does not include the batch dimension).
+ *
+ * @param rate Fraction of the input units to drop. Float between 0 and 1.
```
Comment (hkvision) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras2/layers/Dropout.scala]: Double

**Comment by yjjiang11** [comment]
Jenkins passed after adding padding to MaxPooling2D:http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/778/

**Comment by hkvision** [comment]
LGTM

**Comment by zhichao-li** [comment]
jenkins?

**Comment by hkvision** [comment]
jenkins passed http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/789/
